### ISSUE TITLE
SyntaxError: The requested module '/node_modules/.pnpm/chalk@4.1.2/node_modules/chalk/source/index.js' does not provide an export named 'default'

### ISSUE THREAD

**Issue by sdelpuerto-intelygenz** [lingui/js-lingui#1682]
**Describe the bug**
There is an exception when importing from `@lingui/macro`

**To Reproduce**
Steps to reproduce the behavior, possibly with minimal code sample, e.g:

- Follow the [setup guide for Vite](https://lingui.dev/tutorials/setup-vite#setup-with-vitejs-plugin-react-swc)
- Import anything from `@lingui/macro`

```jsx
import { i18n } from '@lingui/core'
import { I18nProvider } from '@lingui/react'
import { Trans } from '@lingui/macro'
import { messages } from '~/locales/en.po'

i18n.load(locale, messages)
i18n.activate(locale)

export default function App() {
  return (
    <I18nProvider i18n={i18n}>
      <Trans>This should be translated!</Trans>
    </I18nProvider>
  )
}
```

**Expected behavior**
There should not be an exception.

**Additional context**
I'm using PNPM but I get the same error with NPM and Yarn, so it can't be a PNPM bug.

- jsLingui version: `4.2.0`
- Babel version `N/A`
- Macro support:
 - [x] I'm using SWC with `@lingui/swc-plugin`
 - [ ] I'm using Babel with `babel-macro-plugin`
 - [ ] I'm not using macro
- Framework: `Vite`

**Comment by thekip** [comment]
Unfortunately, this information is not enough. Please create a minimal reproduction. The https://github.com/lingui/js-lingui/tree/main/examples/vite-project-react-babel vite example i've tested works well. so it's might be related to other settings of your project.

**Comment by sdelpuerto-intelygenz** [comment]
I'm trying to create an example project but for some reason the SWC plugin isn't working:

```
failed to invoke @lingui/swc-plugin/target/wasm32-wasi/release/lingui_macro_plugin.wasm
```

Could you please try to reproduce this on your end with Vite + React + SWC? The one you tested with is using Babel.

**Comment by sdelpuerto-intelygenz** [comment]
Ok I managed to create a project to reproduce the original issue: [vite-react-swc.zip](https://github.com/lingui/js-lingui/files/11600157/vite-react-swc.zip)

Please take a look if you can.

**Comment by thekip** [comment]
I also reproduced this on my own. Thanks, i will look into this.

**Comment by thekip** [comment]
i've created a working example https://github.com/lingui/js-lingui/pull/1684

I also had the same issue, in Vite it's just a sign that lingui's macro was not processed by plugin in compile time for whatever reason and Vite tries to bundle macro internals which is not supposed to be consumed by browser.

**Comment by sdelpuerto-intelygenz** [comment]
This solves the error I ran into while trying to reproduce the original issue, but the original issue is still there. When I run the project, I get this error in the console:

>Uncaught SyntaxError: The requested module '/node_modules/.pnpm/chalk@4.1.2/node_modules/chalk/source/index.js?v=603d8717' does not provide an export named 'default' (at index.mjs?v=603d8717:1:8)

Please take a look at the project I uploaded in my last comment, you should be able to reproduce that too.

**Comment by thekip** [comment]
@sdelpuerto-intelygenz please read the example and example's readme. I prepared fully working example, just copy it to your project and you're good to go. 

Please don't ask opensource people to review your code, do it yourself, you have all tips already here. 

In your example swc-plugin doesn't work. 

>  it's just a sign that lingui's macro was not processed by plugin in compile time for whatever reason and Vite tries to bundle macro internals which is not supposed to be consumed by browser.

The root cause is: 

> The swc plugin may not working becose of [versions mismatch ](https://github.com/lingui/swc-plugin#compatibility) or this [issue](https://github.com/lingui/swc-plugin/issues/61)

In example's [readme](https://github.com/lingui/js-lingui/tree/main/examples/vite-project-react-swc#swc-compatibility): 

> The version of @swc/core is specified within the @vitejs/plugin-react-swc package.
>
>To ensure that the resolved version of @swc/core is one of the supported versions, this example utilizes the resolutions field in the package.json

**Comment by sdelpuerto-intelygenz** [comment]
I downloaded your example but I'm still getting the same error in the console.

I'm not asking anyone to review my code, I just provided a minimal reproduction like you asked me.

In the example zip file I provided, the SWC plugin works just fine, I used the `resolution` field in the `package.json`.

In summary, I'm running into this error in my actual project, in the minimal reproduction I provided, and in the example you provided. This is a real issue and I think it should be re-opened and looked into.
